src/Concordium/Client/Runner.hs

```diff
@@ -3951,18 +4058,23 @@ processCredential ::
   => BSL.ByteString
   -> Int
   -> ClientMonad m Types.BareBlockItem
-processCredential source networkId =
+processCredential source _networkId =
```

Comment:
Why is the parameter not getting removed alltogether?

Reply:
It wa tempting to do, but I didn't know if this was for a separate PR, so I was a bit wary.

I'll just do it here then?

I do not remember this is the case, or if I perhaps already removed it, but should it be removed from the CLI if it is  there?

Reply:
We just used default 100 I think. This parameter was never used even in the old API.